v0/destinations/vero/transform.js

```diff
@@ -0,0 +1,134 @@
+const set = require("set-value");
+const get = require("get-value");
+const {
+  defaultRequestConfig,
+  CustomError,
+  constructPayload,
+  isDefinedAndNotNull,
+  removeUndefinedAndNullAndEmptyValues,
+  getIntegrationsObj
+} = require("../../util");
+const { EventType } = require("../../../constants");
+const { CONFIG_CATEGORIES, MAPPING_CONFIG } = require("./config");
+
+// This function handles the common response payload for the supported calls
+const commonResponseBuilder = (message, category, destination) => {
+  const payload = constructPayload(message, MAPPING_CONFIG[category.name]);
+  const { authToken } = destination.Config;
+  set(payload, "auth_token", authToken);
+
+  // channels is defined only for Identify call
+  if (category.type === EventType.IDENTIFY) {
+    const channels = get(payload, "channels");
+    const address = get(channels, "address");
```

Comment:
Directly refer from payload no need to declare new variable here reduces code lines

Reply:
Done